ui/packages/consul-ui/app/components/hashicorp-consul/pageobject.js

```diff
@@ -50,7 +50,7 @@ export default (collection, clickable, attribute, is, authForm, emptyState) => s
     ':checked',
     '[data-test-nspace-menu] > input[type="checkbox"]'
   );
-  page.navigation.dcs = collection('[data-test-datacenter-menu] li', {
+  page.navigation.dcs = collection('[data-test-datacenter-menu] [data-test-dc-item]', {
```

Comment:
I was wondering why the `li` didn't work here and spotted we've added some erroneous HTML:

https://github.com/hashicorp/consul/blob/bea497b97ab1c1a1060ac6eaae487a40d2125716/ui/packages/consul-ui/app/components/consul/datacenter/selector/index.hbs#L59

^ That means we have:

```html
<ul>
  <li>
    <li>dcname</li>
  </li>
</ul>
```
...so an li directly inside an li, which I don't think is allowed.

As this is all related to https://github.com/hashicorp/consul/pull/13753 could this also be fixed up here? (sorry I missed this in the original review)

(P.S. I don't think this spot answers the question of the `li` not working btw, but I was just curious about that more than anything)

Reply:
The separator also will display as a `li` in the list. that's the reason for the change.

Reply:
oh! Of course 😆 thankyou!